packages/flame/lib/src/events/messages/drag_start_event.dart

```diff
@@ -34,4 +34,9 @@ class DragStartEvent extends PositionEvent {
       ),
     );
   }
+
+  @override
+  String toString() =>
+      'DragStartEvent(pointerId: $pointerId, deviceKind: $deviceKind, '
+      'canvasPosition: $canvasPosition, devicePosition: $devicePosition)';
```

Comment:
The `devicePosition` is rarely useful, since the games almost always run in fullscreen mode. Instead, let's replace it with the `localPosition`. Also, `canvasPosition`/`localPosition` are more important than `pointerId`/`deviceKind`, so let's switch them around.